src/pages/Movie/TopInfo/component.js

```diff
@@ -107,23 +110,27 @@ const TopInfo = ({
 );
 
 TopInfo.propTypes = {
+  id: PropTypes.number,
   overview: PropTypes.string,
   title: PropTypes.string,
   originalLanguage: PropTypes.string,
   runtime: PropTypes.number,
   budget: PropTypes.number,
   revenue: PropTypes.number,
-  genres: PropTypes.arrayOf(PropTypes.object)
+  genres: PropTypes.arrayOf(PropTypes.object),
+  isMovieInWatchlist: PropTypes.bool
```

Comment:
Is it correct that you might have movie without this value?

Reply:
It would be better to discuss it on the daily.

Reply:
Fixed